tests/app/test_jinja_filters.py

```diff
@@ -1,26 +1,46 @@
 from datetime import datetime
 from unittest import TestCase
 
-from app.jinja_filters import pretty_date
-from app.jinja_filters import pretty_date_range
+from app.jinja_filters import format_date
+from app.jinja_filters import format_str_as_date_range
+from app.jinja_filters import format_str_as_date
+from app.jinja_filters import format_str_as_month_year_date
 
 
 class TestJinjaFilters(TestCase):
 
-    def test_format_pretty_date(self):
+    def test_format_date(self):
         # Given
         date = datetime.strptime('01/01/17', '%d/%m/%y')
 
         # When
-        pretty_value = pretty_date(date)
+        format_value = format_date(date)
 
-        self.assertEquals(pretty_value, '1 January 2017')
+        self.assertEquals(format_value, '1 January 2017')
```

Comment:
Why does this give full month name and the others are short month names. maybe the filter names should suggest what you will get?